docs/src/main/asciidoc/getting-started-guide.adoc

```diff
@@ -141,7 +156,7 @@ public class GreetingResource {
     @GET
     @Produces(MediaType.TEXT_PLAIN)
     public String hello() {
-        return "hello\n";
+        return "hello";
```

Comment:
I'm not sure I agree with this chance: the `\n` was added by a contributor because when doing simple command line calls, your shell didn't properly start on a new line without that.

Reply:
That user in https://github.com/quarkusio/quarkus/issues/1987 most likely used quite old style shell setup ;) 

Anyways that’s why I added the curl TIP for those unfortunate.